package.json

```diff
@@ -74,6 +74,11 @@
     "redux": "^3.6.0",
     "redux-thunk": "^2.1.0",
     "reqwest": "^2.0.5",
+    "scribe": "guardian/scribe#fcf4e16",
```

Comment:
Because scribe hasn't been released for a while so newer versions are not available on npm. I could look into releasing it later today.